sources/packages/backend/apps/db-migrations/src/sql/Reports/Create-data-inventory-report.sql

```diff
@@ -0,0 +1,505 @@
+INSERT INTO
+    sims.report_configs (report_name, report_sql)
+VALUES
+    (
+        'Data_Inventory_Report',
+        'select
+            -- Personal Info
+            users.first_name as "First Name",
+            users.last_name as "Last Name",
+            students.birth_date as "Date of Birth",
+            students.gender as "Gender",
+            sin_validations.sin as "SIN",
+            users.email as "Email address",
+            students.contact_info -> ''address'' ->> ''addressLine1'' as "Address Line 1",
+            students.contact_info -> ''address'' ->> ''addressLine2'' as "Address Line 2",
+            students.contact_info -> ''address'' ->> ''city'' as "City",
+            students.contact_info -> ''address'' ->> ''provinceState'' as "Province",
+            students.contact_info -> ''address'' ->> ''postalCode'' as "Contact Postal/Zip Code",
+            students.contact_info -> ''address'' ->> ''country'' as "Country",
+            students.contact_info ->> ''phone'' as "Contact Phone Number",
+            -- Account Info
+            (
+                select
```

Comment:
Is it ok to be not `N` otherwise?

Reply:
It wasn't asked for 'N' but I changed it.